metastore/scripts/upgrade/hive/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.hive.sql

```diff
@@ -0,0 +1,887 @@
+SELECT 'Upgrading MetaStore schema from 3.1.0 to 4.0.0-alpha-1';
```

Comment:
Shouldn't this Hive upgrade script be mostly empty (except version update) at the moment?

Also the query should be the following I think:
`SELECT 'Upgrading MetaStore schema from 4.0.0-alpha-1 to 4.0.0-alpha-2';`